.github/workflows/macos.yml

```
@@ -86,12 +143,116 @@ jobs:
           echo "TESTS=${TESTS}" >> $GITHUB_ENV
         if: ${{ matrix.test_task == 'check' && matrix.skipped_tests }}
 
-      - name: Set up Launchable
-        id: enable-launchable
-        uses: ./.github/actions/launchable/setup
+      - uses: actions/download-artifact@v4
+        with:
+          name: ${{ matrix.os }}${{ matrix.test_task }}
+        if: ${{ always() && (matrix.test_task == 'check' || matrix.test_task == 'test-all') }}
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+        if: ${{ always() && (matrix.test_task == 'check' || matrix.test_task == 'test-all') }}
+      - name: Install launchable command
+        run: |
+          pip install launchable
+          launchable split-subset --subset-id $(cat ../launchable-subset-id.txt) --bin 1/4 raw > launchable-subset.txt
+          echo "TESTS=$(cat launchable-subset.txt | tr '\n' ' ' | sed 's/file=/..\/src\//g')" >> $GITHUB_ENV
```

`sed` applies the command for each line, and `$(...)` replaces `\n` with a space like `tr` does here.
```suggestion
          echo "TESTS=$(cat launchable-subset.txt | sed 's:file=:../src/:')" >> $GITHUB_ENV
```